File: java-course/src/test/java/com/example/selenium/spree/pages/CartPage.java
package com.example.selenium.spree.pages;

import org.openqa.selenium.remote.RemoteWebDriver;

public class CartPage extends BasePage {
    public static final String URL = "https://spreecommerce-demo.herokuapp.com/cart";

    private static final String TITLE = "Shopping Cart - Spree Demo Site";

    public CartPage(RemoteWebDriver webDriver) {
        super(webDriver, TITLE, URL);
    }
}

File: python-course/com/example/selenium/spree/pages/CartPage.py
from com.example.selenium.spree.pages.BasePage import BasePage

class CartPage(BasePage):
    title = "Shopping Cart - Spree Demo Site"
    url = "https://spreecommerce-demo.herokuapp.com/cart"

    def __init__(self, test):
        BasePage.__init__(self, test, self.title, self.url)
        self.test = test
